files/scripts/arp_update

```diff
@@ -46,5 +46,24 @@ while /bin/true; do
       ip6cmd="ip -6 neigh show | grep -v fe80 | grep $vlan | cut -d ' ' -f 1,3 | $ndisc6cmd"
       eval `eval $ip6cmd`
   done
+
+  # refresh neighbor entries from APP_DB in case of mismatch with kernel
+  DBNEIGH=$(redis-cli -n 0 -c keys *NEIGH_TABLE*)
```

Comment:
this command is more efficient.

```
admin@vlab-01:~$ redis-cli --scan --pattern NEIGH_TABLE* 
NEIGH_TABLE:eth0:10.250.0.1
admin@vlab-01:~$ 
```

Reply:
sure, i'll change it to this and resubmit